calico_containers/pycalico/datastore.py

```diff
@@ -137,17 +139,55 @@ class DatastoreClient(object):
     """
 
     def __init__(self):
+        etcd_endpoints = os.getenv(ETCD_ENDPOINTS_ENV, ETCD_ENDPOINTS_DEFAULT)
```

Comment:
I've gone with what matches the other calls a few lines below (and throughout most of the rest of Calico), to which the same argument could apply.